metpy/calc/thermo.py

```diff
@@ -222,8 +222,8 @@ def lfc(pressure, temperature, dewpt):
     parcel_profile
     """
     ideal_profile = parcel_profile(pressure, temperature[0], dewpt[0]).to('degC')
-    x, y = find_intersections(pressure, ideal_profile, temperature)
-    return x[0], y[0]
+    x, y = find_intersections(pressure[1:], ideal_profile[1:], temperature[1:])
+    return (x[0], y[0])
```

Comment:
No parens--I can't believe I missed that in the last PR. I'm terrible at code review.